src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -333,7 +333,6 @@ private async Task SendHeartBeatAsync()
                 _currentWriter.Write("{\"Type\":\"HeartBeat\"}");
 
                 await FlushIfNeeded().ConfigureAwait(false);
```

Comment:
Actually, this should be: `FlushIfNeeded(force: true)`.
and inside:
`force =  force || IsHeartbeatIntervalExceeded();`